lib/cocoapods-core/cdn_source.rb

```diff
@@ -68,6 +69,21 @@ def preheat_existing_files
       end
     end
 
+    def files_definitely_to_update
+      [repo.join('**/*.yml'), repo.join('**/*.txt')].
```

Comment:
use `**/*.{yml,txt}`? This will invoke it once and could be faster and also no need to invoke `flatten`